rskj-core/src/test/java/co/rsk/mine/MineBlockWithUncleTest.java

```diff
@@ -43,36 +43,31 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.List;
 
+import static org.ethereum.crypto.HashUtil.EMPTY_TRIE_HASH;
+import static org.ethereum.util.ByteUtil.EMPTY_BYTE_ARRAY;
 import static org.hamcrest.Matchers.empty;
-import static org.junit.Assert.*;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
 import static org.mockito.Mockito.*;
 
 @RunWith(PowerMockRunner.class)
-@PrepareForTest(FamilyUtils.class)
-public class BlockMineTest {
+@PrepareForTest({FamilyUtils.class})
+public class MineBlockWithUncleTest {
```

Comment:
Change the name of this class to `BlockToMineBuildetTest.java` to comply with tests classes naming of the project.

Reply:
The test sets up a very specific context which won't really extend to other `BlockToMineBuilder` tests, and I suggested the name.
I guess it's a matter of preference but I thought I'd leave this here for reference.

Reply:
I think it would apply to other `BlockToMineBuilder` tests since with the mocks it covers the whole flow of block creation.